src/main/java/com/massivecraft/factions/integration/LWCFeatures.java

```diff
@@ -0,0 +1,72 @@
+package com.massivecraft.factions.integration;
+
+import com.griefcraft.lwc.LWC;
+import com.griefcraft.lwc.LWCPlugin;
+import com.massivecraft.factions.*;
+import org.bukkit.Bukkit;
+import org.bukkit.Chunk;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockState;
+import org.bukkit.plugin.Plugin;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class LWCFeatures {
+    private static LWC lwc;
+
+    public static void setup() {
+        Plugin test = Bukkit.getServer().getPluginManager().getPlugin("LWC");
+        if (test == null || !test.isEnabled()) return;
```

Comment:
Maybe swap out the `== null` for `instanceof LWCPlugin` to do both a nullcheck *and* confirm it's not going to somehow result in a cast exception?